lib/internal/test_runner/runner.js

```diff
@@ -593,6 +593,7 @@ function run(options = kEmptyObject) {
     argv = [],
     cwd = process.cwd(),
     rerunFailuresFilePath,
+    hookSignal = false,
```

Comment:
If we decide to add a new property to the run method, we also need to update the documentation:
https://github.com/nodejs/node/blob/fb614c43240158fa6c5b988571746fadff8e22d7/doc/api/test.md?#runoptions